themes/digital.gov/src/scss/new/guides/_guide-menu-bar.scss

```diff
@@ -78,3 +78,101 @@
     }
   }
 }
+
+.dg-accordion-mobile-menu {
+  position: sticky;
+  top: -1px;
+  z-index: z-index(100);
+
+  &__content {
+    background-color: color("gray-3");
+    padding: 0;
+  }
+
+  &__heading {
+    align-items: center;
+    background-color: color("gray-cool-5");
+    display: flex;
+    flex-direction: row;
+    height: 3.25rem;
```

Comment:
I'm seeing a bug on iOS mobile. Testing with Browsterstack works for Chrome, Firefox and Edge (not pictured).

<img width="924" alt="Screenshot 2024-04-18 at 10 43 32 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/f78e5d4d-75c1-4f50-b2c3-fb8f00d59535">

Reply:
Fixed this by setting `width: auto` to `dg-accordion-mobile-menu__image-glossary` and `dg-accordion-mobile-menu__image-glossary svg`.